gdp-src-build/conf/templates/renesas-rcar-gen3.local.inc

```diff
@@ -0,0 +1,143 @@
+# Include generic GDP local configuration
+include local.inc
+
+# Add R-Car Gfx and MMP packages to install
+IMAGE_INSTALL_append_rcar-gen3 = "libdrm-kms kernel-module-gles gles-user-module libgbm-dev packagegroup-gstreamer1.0-plugins weston-bin kernel-module-pvrsrvkm"
+
+MACHINE ?= "m3ulcb"
+
+# This sets the SoC
+# H3: r8a7795, M3: r8a7796
+#SOC_FAMILY = "r8a7795"
+SOC_FAMILY ?= "r8a7796"
+
+# Add systemd configuration
+DISTRO_FEATURES_append = " systemd"
+VIRTUAL-RUNTIME_init_manager = "systemd"
```

Comment:
@chbae Hi I would say then lets update README.md once the PR is settled as the SHA keeps changing as a result of change requests.